src/selectedtests/test_mappings/create_test_mappings.py

```diff
@@ -68,8 +86,8 @@ def generate_project_test_mappings(
     temp_dir: TemporaryDirectory,
     source_re: Pattern,
     test_re: Pattern,
-    after_date: datetime,
-):
+    commit_limit: str,
```

Comment:
Should `commit_limit` be a `CommitLimit`?

It looks like it is typed as a `str` in several places.

Reply:
good catch 👍